pom.xml

```
@@ -85,7 +85,7 @@
   <org.slf4j.version>1.7.30</org.slf4j.version>
   <com.google.code.gson.version>2.8.9</com.google.code.gson.version>
   <com.github.ben-manes.caffeine.version>3.0.1</com.github.ben-manes.caffeine.version>
-  <org.hibernate.hibernate.version>5.6.7.Final</org.hibernate.hibernate.version>
+  <org.hibernate.hibernate.version>5.6.14.Final</org.hibernate.hibernate.version>
```

```suggestion
  <org.hibernate.hibernate.version>5.6.12.Final</org.hibernate.hibernate.version>
```
From:
https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L91